aries-site/src/pages/components/datatable.mdx

```diff
@@ -434,6 +449,19 @@ Should be guided by the task the user is setting out to accomplish.
 - Do rows have many interactive elements? --> Pagination
   - Be mindful of keyboard users.
 
+### Vertically align row data
+
+When a column has multi-line data, it may improve readability to align the data to the top of the row. In these cases, apply `verticalAlign={{ body: 'top' }}` to the DataTable.
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/datatable/DataTableVerticalListExample.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
```

Comment:
Let's add the `gitHub` prop and link to where this file will live in the repo. This allows the user to view the code in context with the rest of the project should they choose, while the `code` prop allows them to inspect the code within the page.